### ISSUE TITLE
Format the SveltyPicker Button

### ISSUE THREAD

**Issue by nicholasvperry** [mskocik/svelty-picker#81]
I see you can format the calendar once it is open, but I cannot seem to style the actual button that you click to open the calendar. I cannot put a boarder on it or change font/background.
![image](https://user-images.githubusercontent.com/90103179/214950377-4c18fc34-6831-4148-9ea7-f4f3e6085b62.png)

![image](https://user-images.githubusercontent.com/90103179/214950313-0a1867bd-3a44-4547-a078-8e19159b09bc.png)

**Comment by mskocik** [comment]
It's not a button, it's regular input. You can also pass CSS class for 
input as `inputClasses` property

**Comment by mskocik** [comment]
That's how styles works in svelte. You need also `calendar` to be defined global

**Comment by nicholasvperry** [comment]
Awesome! Thanks for your help! I appreciate it. I have attached a picture for anyone else who may have been having the same issue as me. I also noticed that if I did not put the my-colors style that the calendar is see through and that is why I have them in my style.

![image](https://user-images.githubusercontent.com/90103179/215125324-e0e79764-f212-4dae-baf7-ca6afb0eb12b.png)
![image](https://user-images.githubusercontent.com/90103179/215125724-a7c21b64-8be3-42e8-a7a4-ed1bb655d5ba.png)
